ReportPortal.ATM/TestProject/Pages/AllDashboardPage/AllDashboardsPage.cs

```diff
@@ -0,0 +1,50 @@
+﻿using OpenQA.Selenium;
+using System;
+using WebDriverLibrary.Interfaces.WebDrivers;
+
+namespace TestProject.Pages.AllDashboardPage;
+
+public partial class AllDashboardsPage
+{
+    private readonly IWebDriverService _webDriverService;
+    private readonly IWebDriver _webDriver;
+
+    public AllDashboardsPage(IWebDriverService webDriverService)
+    {
+        ArgumentNullException.ThrowIfNull(webDriverService);
+
+        _webDriverService = webDriverService;
+        _webDriver = _webDriverService.GetWebDriver();
+    }
+
+    public void OpenCreateNewDashboard()
+    {
+        ClickOnAddNewDashboard();
+    }
+
+    public void OpenDashboard(string name)
+    {
+        ArgumentException.ThrowIfNullOrWhiteSpace(name);
+
+        ClickOnDashboard(name);
+    }
+
+    public void OpenEditDashboard()
+    {
+        ClickOnEditDashboard();
+    }
+
+    public void CheckDashboardName(string name)
+    {
+        ArgumentException.ThrowIfNullOrEmpty(name);
+
+        Assert.That(FindDashboardName(name).Displayed, Is.EqualTo(true));
```

Comment:
Assertions should only be used at the SpecFlow Step level, the assertions are part of the test runner framework we are using.

If we use assertions inside the POM, we will be creating a tightly-coupled relation between the POM and the Test Runner, and the POM should always be agnostic to whatever Test Runner we decide to use.

The POM should only allow us to interact with the page and to retrieve data from the page, then, in the test step, we should assert the actual vs the expected result of that interaction or the retrieved data.